src/main/java/SELayer.mdl

```diff
@@ -326,6 +326,9 @@ type SETitle extends PDStructElem {
 type SEFENote extends PDStructElem {
     % value of the NoteType attribute
     property NoteType : String;
+    % comma separated list of object numbers, that the Ref entry in this FENote does not contain 
```

Comment:
% comma separated list of object numbers that are missing in the Ref entry of a given FENote